src/features/Header/views/Header.jsx

```diff
@@ -0,0 +1,5 @@
+function Header() {
```

Comment:
Are u sure that we need to put header inside features?
Maybe it would be better to put inside components out of features if u will reuse it, or if no put it inside CurrentCityForecastFeature inside comonents

Reply:
It's not reusing component and this component will be on all pages.

Reply:
move it to folder src/features/components/Header.